lib/puppet-lint/bin.rb

```diff
@@ -76,7 +92,17 @@ def run
           fd.write(l.manifest)
         end
       end
-      puts ']' if PuppetLint.configuration.json
+
+      if PuppetLint.configuration.sarif
+        report_sarif(all_problems, full_base_path, full_base_path_uri)
+      elsif PuppetLint.configuration.json
+        all_problems.each do |problems|
+          problems.each do |problem|
+            problem.delete_if { |key, _| [:description, :help_uri].include?(key) }
```

Comment:
I'm not sure the description must be excluded. The help URI also doesn't hurt IMHO. Perhaps a maintainer can weigh in on that.

Reply:
Perhaps a maintainer can weigh in on that. ---- yes the only reason for this code block is I am trying to maintain the same JSON output as before, maintainer can advise if also want these 2 new properties in JSON and let me know to change accordingly.